src/test/java/api/loans/scenarios/RefundAgedToLostFeesTestBase.java

```diff
@@ -42,7 +37,9 @@
   private final String cancellationReason;
 
   protected RefundAgedToLostFeesTestBase(String cancellationReason) {
-    this.cancellationReason = cancellationReason;
+//    this.cancellationReason = cancellationReason;
+    // FIXME: uncomment when API will be updated and cancellation reason will be possible to set
+    this.cancellationReason = "Cancelled as error";
```

Comment:
Is there a tech debt issue for resolving this once the fee/fines APIs have been changed?

Are the POs aware of this temporary change in behaviour?

Reply:
This will be blocked until the API is fixed.

Reply:
Ok, thanks